.github/workflows/ci.yml

```
@@ -36,6 +36,34 @@ jobs:
       - name: Run tests
         run: make test-${{ matrix.variation }}
 
+  wasm:
+    name: SwiftWasm
+    runs-on: ubuntu-latest
+    env:
+      OMIT_MACRO_TESTS: 1
```

```suggestion
    env:
      OMIT_MACRO_TESTS: 1
```
None to worry about in this repo, right?